ui/src/components/LeftNav/MapLayerRow.vue

```diff
@@ -121,6 +121,7 @@ li:hover {
   border: 1px;
   padding: 2px 3px;
   font-size: 12px;
+  /*font-size: 1em;*/
```

Comment:
leftover comment?

Reply:
Yes. However, moving away from "px" units to relative units like "em" or "rem" is an important but future topic.